apps/pie-docs/src/patterns/patterns.json

```diff
@@ -0,0 +1,4 @@
+{
+  "ogImage": "landing_accessibility.png",
```

Comment:
Just double checking - do we want a new illustration here?

Reply:
I wasn't aware of this one! I will choose maybe for now the components image and double check with Molly what she would like to use and bring it later on so this doesn't block this pr :)